.github/workflows/build_publish_openmrs.yml

```diff
@@ -20,8 +21,6 @@ jobs:
       HELM_CHART_PATH: bahmni-emr/package/helm/openmrs
```

Comment:
can we remove HELM_CHART_PATH as well, since all the steps related helm charts are removed?

Reply:
This needs to be resolved.